.github/workflows/nightly.yml

```diff
@@ -22,8 +22,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-        with:
```

Comment:
This is needed for the 1.20.6+ versioning scheme as this pulls the git tag down for reading (best way I could find to get checkout to give me tags)